bindings/node/lib/cryptoCallbacks.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+const crypto = require('crypto');
+
+function aes256CbcEncryptHook(key, iv, input, output) {
+  let result;
+
+  try {
+    let cipher = crypto.createCipheriv('aes-256-cbc', key, iv);
+    cipher.setAutoPadding(false);
+    result = cipher.update(input);
+  } catch (e) {
+    console.dir({ e });
```

Comment:
do we want there to be `console.dir` here? seems like something you'd want to pass to whatever the `logger` is?

Reply:
Created [NODE-2082](https://jira.mongodb.org/browse/NODE-2082) to track this work